### PR TITLE
Map/table interaction funkiness

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes issue(s) #1901 

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/open-sesame/)

Currently the only browser that it is possible to reliably open the county table by clicking on a county is **my own chrome browser** :(. Currently, the new set of fixes here doesn't _completely_ fix it, but it is now possible for @ericronne to occasionally open his table by clicking on a county

MORE TO COME

### CODE REVIEW

**Comment by ericronne** [comment]
Worx for me!!

**Comment by gemfarmer** [comment]
@ericronne cool! I'm going to do a little clean up, then should be good to go!

File: js/components/eiti-data-map-table.js [hunk 1]
```diff
@@ -131,15 +141,14 @@
...
-        toggleTable(this);
-      })
-      .on('mouseover.county', function(){
-        var that = this;
-        setTimeout(function(){
-          eiti.util.throttle(mouseTable(that, 'mouseover'), 200);
-        }, 10);
-      });
+      .on('mouseover.county', mouseTable)
+      .on('click.countyClick', toggleTable)
```
Comment (gemfarmer) [inline js/components/eiti-data-map-table.js]: no longer an issue

File: js/components/eiti-data-map-table.js [hunk 2]
```diff
@@ -80,19 +85,23 @@
...
+    var mouseTable = function() {
+      var event = event || d3.event || window.event;
+      var eventType = event.type;
+      var context = this || event.target || event.srcElement;
+      console.log(eventType)
 
       var parent = context.parentNode;
       var hasValue = parent.getAttribute('data-value') &&
         parent.getAttribute('data-value') !== 'null';
+      console.log(hasValue, parent)
```
Comment (gemfarmer) [inline js/components/eiti-data-map-table.js]: console has been removed

**Pull request merged**